src/webgpu/api/operation/command_buffer/image_copy.spec.ts

```diff
@@ -26,7 +26,6 @@ export const description = `writeTexture + copyBufferToTexture + copyTextureToBu
 * TODO:
   - add another initMethod which renders the texture [3]
   - test copyT2B with buffer size not divisible by 4 (not done because expectContents 4-byte alignment)
-  - add tests for 1d / 3d textures
```

Comment:
I think this file is still just 2d, unless we've decided to not test 1d/3d here. Or maybe you have the modifications locally and accidentally added this line to the commit.